src/components/BuySellForm/BuySellData/BuySellData.tsx

```diff
@@ -1,9 +1,11 @@
 import { forwardRef } from 'react';
+import clsx from 'clsx';
 import { THooks } from 'types';
 import { PaymentMethodWithIcon } from '@/components';
+import { RATE_TYPE } from '@/constants';
 import { formatTime } from '@/utils';
 import { Localize, useTranslations } from '@deriv-com/translations';
-import { Text, useDevice } from '@deriv-com/ui';
+import { Text, Tooltip, useDevice } from '@deriv-com/ui';
```

Comment:
FYI this tooltip version has bugs and @aizad-deriv created a new one it will release in the next version the props of the new version is different from this one

Reply:
alright, we'll make sure to migrate it properly when it gets released 🙏